hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -51,6 +52,7 @@ import (
 	k8sutilspointer "k8s.io/utils/pointer"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
This is already imported on line 54.  Lets not find/replace the name of this import i.e. use `client` not `ctrlclient`.  This was recently changed in https://github.com/openshift/hypershift/pull/419 so maybe this is just a rebase issue.

Reply:
you're right